pkg/minishift/addon/addon_meta.go

```diff
@@ -60,6 +64,11 @@ func NewAddOnMeta(headers map[string]interface{}) (AddOnMeta, error) {
 			return nil, nil
 		}
 	}
+	if headers[RequiredOpenShiftVersion] != nil {
+		if !checkVersionSemantic(headers[RequiredOpenShiftVersion].(string)) {
+			return nil, errors.New("Add-on only support OpenShift version semantics eg. 3.6.0 or >3.6.0, <3.9.0 or >=3.5 etc.")
```

Comment:
@hferentschik Does that error message is good enough for the user instead of providing document link? we can reword it if you think something else should be added.